config/zones/CO.yaml

```diff
@@ -1,2 +1,15 @@
 flag_file_name: co.png
```

Comment:
```suggestion
```
This is not really related to your pr but this key is no longer used in the frontend so let's remove it 🙂